docs/source/tutorials/outliers.ipynb

```diff
@@ -2,46 +2,75 @@
     "- Create `train_data` and `test_data` such that `train_data` only contains images of animals and `test_data` contains images from all classes.\n",
     "- Load pretrained `timm` neural network model and extract `train_data` and `test_data` feature embeddings from its representations.\n",
     "- Use `cleanlab` to find naturally occurring outlier examples in the `train_data` (i.e. atypical images).\n",
     "- Find outlier examples in the `test_data` that do not stem from training data distribution (including out-of-distribution non-animal images).\n",
-    "- Explore threshold selection for determining which images are outliers vs not."
+    "- Explore threshold selection for determining which images are outliers vs not.\n",
+    "\n",
+    "Detecting outliers using `pred_probs` from a trained classifier\n",
+    "- Turn our `timm` model into a classifier by adding final output layer fit via Multi-Layer Perceptron to the (in-distribution) training data.\n",
+    "- Use `cleanlab` to find naturally occuring OOD examples in the dataset."
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "41552bff",
+   "id": "70016f64",
    "metadata": {},
    "source": [
     "<div class=\"alert alert-info\">\n",
     "Quickstart\n",
     "<br/>\n",
     "    \n",
-    "Already have numeric feature embeddings for your data? Just run the code below to compute outlier scores.\n",
+    "Already have numeric **feature embeddings** for your data? Just run the code below to compute out-of-distribution feature scores.\n",
     "\n",
     "\n",
     "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">  \n",
     "    \n",
     "```python\n",
     "\n",
-    "from cleanlab.rank import get_outlier_scores\n",
+    "from cleanlab.outlier import OutOfDistribution\n",
     "    \n",
+    "OOD = OutOfDistribution()\n",
+    "\n",
     "# To get outlier scores for train_data using feature matrix train_feature_embeddings\n",
-    "outlier_scores, knn = get_outlier_scores(features=train_feature_embeddings, return_estimator=True)\n",
+    "ood_train_feature_scores = OOD.fit_score(features=train_feature_embeddings)\n",
     "\n",
     "# To get outlier scores for additional test_data using feature matrix test_feature_embeddings\n",
-    "test_outlier_scores = get_outlier_scores(features=test_feature_embeddings, knn=knn)\n",
+    "ood_test_feature_scores = OOD.score(features=test_feature_embeddings)\n",
+    "    \n",
+    "    \n",
+    "```\n",
+    "\n",
+    "</div>\n",
+    "    \n",
+    "Already have `pred_probs` and `labels` for your data? Just run the code below to compute out-of-distribution predictions scores.\n",
+    "\n",
+    "\n",
+    "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">  \n",
+    "    \n",
+    "```python\n",
+    "\n",
+    "from cleanlab.outlier import OutOfDistribution\n",
+    "    \n",
+    "OOD = OutOfDistribution()\n",
```

Comment:
I think using capitalized variable name is nonconventional?

https://peps.python.org/pep-0008/#:~:text=Prescriptive%3A%20Naming%20Conventions,-Names%20to%20Avoid&text=Never%20use%20the%20characters%20'l,the%20numerals%20one%20and%20zero.

Reply:
changing to ood